reference-implementation/lib/exclusive-stream-reader.js

```diff
@@ -11,17 +12,17 @@ export default class ExclusiveStreamReader {
       throw new TypeError('This stream has already been locked for exclusive reading by another reader');
     }
 
-    stream._readableStreamReader = this;
+    AttachExclusiveStreamReader(stream, this);
```

Comment:
I gave it its own operation name for readability. But maybe just adding comments is fine. Reverted.